BlueprintUI/Sources/Layout/EqualStack.swift

```diff
@@ -0,0 +1,118 @@
+/// An element that sizes all of its children equally, stacking them according to
+/// the specified `direction` and spacing them according to the specified `spacing`.
```

Comment:
What's the cross-axis layout story now?

Reply:
Always fill

Reply:
Sorry, should have been clearer: can we add the cross-axis layout behavior to the documentation?

Reply:
I already updated the note to (try to) document the behavior for both the primary/secondary axis:

```swift
/// - Note: A stack measures itself by determining its largest child in each axis, and
///         in the case of the primary axis, multiplying by `children.count` (accounting
///         for `spacing` as necessary).
```

if this isn't explicit enough, I could also add something along the lines of:

```swift
/// In the secondary axis, the stack will fill the space occupied by its largest child.
```

let me know!

Reply:
I see. I guess the way that reads to me, it only explains the measuring. I don't think it would hurt to be explicit about how children will be justified to the full size in the cross-axis.

Reply:
here's what I went with:

```swift
/// An element that sizes its children equally, stacking them in the primary axis according to
/// the specified `direction` and spacing them according to the specified `spacing`. In the
/// secondary axis, children are justified to fill the space of the largest child.
///
/// - Note: A stack measures itself by determining its largest child in each axis, and
///         in the case of the primary axis, multiplying by `children.count` (accounting
///         for `spacing` as necessary).
```